pytorch_lightning/metrics/compositional.py

```diff
@@ -85,8 +85,8 @@ def persistent(self, mode: bool = False):
 
     def __repr__(self):
         repr_str = (
-            self.__class__.__name__
-            + f"(\n  {self.op.__name__}(\n    {repr(self.metric_a)},\n    {repr(self.metric_b)}\n  )\n)"
+            self.__class__.__name__ +
+            f"(\n  {self.op.__name__}(\n    {repr(self.metric_a)},\n    {repr(self.metric_b)}\n  )\n)"
```

Comment:
In my opinion, it looks better without this change here. We could consider adding `split_before_arithmetic_operator = true` to the yapf config to address this.

yapf docs: https://github.com/google/yapf

Reply:
good point added :]